libs/data-mapper/src/lib/components/functionList/FunctionList.tsx

```diff
@@ -154,9 +157,40 @@ export const FunctionList = () => {
     return newFunctionListTree;
   }, [functionData, searchTerm, inlineFunctionInputOutputKeys]);
 
+  const searchLoc = intl.formatMessage({
+    defaultMessage: 'Search',
+    description: 'Search',
+  });
+
+  const clearLoc = intl.formatMessage({
+    defaultMessage: 'Clear',
+    description: 'Clear',
+  });
+
   return (
     <>
-      <TreeHeader onSearch={setSearchTerm} onClear={() => setSearchTerm('')} />
```

Comment:
NOTE: Un-using the abstraction here in favor of specializing the schema tree one (without trying to conditional-ize stuff) - figured it's an okay tradeoff as FunctionList is pretty simple, and it'd be extremely low effort to abstract the new stuff here if we did need a reusable *simple* searchbar (i.e. this) in the future